src/js/components/TaskListItemComponent.jsx

```diff
@@ -300,6 +301,9 @@ var TaskListItemComponent = React.createClass({
           <br />
           {endpoints}
         </td>
+        <td>
+          {host}
```

Comment:
Would it be interesting to link this to https://consul-info-html.par.preprod.crto.in/consul-nodes-ui.html?node_filter=mesos-slave274 ?

Reply:
I'll check if this is easy to do; if not I'll do it in another review :)

Reply:
Done; thanks for the feedback! :)